src/shared/remoteSourceTrackingService.ts

```diff
@@ -538,11 +558,15 @@ export class RemoteSourceTrackingService extends ConfigFile<RemoteSourceTracking
       )
       .map((member) => {
         getMetadataKeyFromFileResponse(member)
-          // CustomObject could have been added by the key generator
           // remove some individual members known to not work with tracking even when their type does
           .filter(
             (key) =>
-              !key.startsWith('CustomObject') && key !== 'Profile__Standard' && key !== 'CustomTab__standard-home'
+              // CustomObject could have been re-added by the key generator from one of its fields
+              !key.startsWith('CustomObject') &&
+              key !== 'Profile__Standard' &&
+              key !== 'CustomTab__standard-home' &&
+              key !== 'AssignmentRules__Case' &&
+              key !== 'ListView__CollaborationGroup.All_ChatterGroups'
```

Comment:
Could maintain an array of these instead like:
const snowflakes = ['Profile__Standard', 'CustomTab__standard-home', 'AssignmentRules__Case', 'ListView__CollaborationGroup.All_ChatterGroups'];
... `!key.startsWith('CustomObject') && !snowflakes.includes(key)`